### DISCUSSION TITLE
Current usage of struts 1.5.0-RC2

### DISCUSSION THREAD

**Discussion by giev02** [ste-gr/struts1-reloaded#41] [General]
Hi,

Are there already productive applications of Struts 1.5.0--RC2 ?

We are interested in using it for some legacy projects.

Best regards,
Ulrich Gievers

**Comment by ste-gr** [comment]
Hi @giev02,

as I know, there are some people using it.
Have a look at other discussions and issues. I think all thread opener are also users for the 1.4 and/or 1.5-version.

Greetings
Stefan

**Comment by ste-gr** [comment]
Hi @giev02,

as I forgot:
In our company we use 1.4.4/1.4.5 since more than a year for a production-application.

**Enviroment:**
* JBoss 7.4
* OpenJDK 11

Greetings
Stefan

**Comment by nrmnrm** [comment]
we are using 1.5RC2 on JBoss 8 / and Wildfly supporting EE10 / jakarta namespace. No issues so far for us
